sharness/lib/test-lib.sh

```diff
@@ -66,6 +68,9 @@ test_cluster_init() {
         echo "error initializing ipfs cluster"
         exit 1
     fi
+    if [[ -n "$custom_config_files" ]]; then
```

Comment:
And `sh` seems sufficient for this, and is also used in the chriscool/sharness example (maybe just to be shell-agnostic though, not sure).